script.rb

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+require_relative 'parser'
+require 'benchmark'
```

Comment:
Delete redundant require

Reply:
why do you talk to youself? man, is everythig okay? LGTM?

Reply:
Haha, hello Dmitriy! :) Yep, i like make reviews. I can miss some defects, so self review help me.

Reply:
Maybe you want to do some review of my code? I would appreciate.